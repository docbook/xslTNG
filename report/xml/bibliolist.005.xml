<?xml version="1.0" encoding="utf-8"?>
<article xmlns="http://docbook.org/ns/docbook" version="5.0"
         xmlns:xlink="http://www.w3.org/1999/xlink">
<info>
<title>Unit Test: bibliolist.005</title>
</info>

<para>Tests for lists of names.</para>

<bibliolist>
<biblioentry>
  <author><personname>Jane Smith</personname></author>
  <title>A book with one author</title>
</biblioentry>

<biblioentry>
  <authorgroup>
  <author><personname>Jane Smith</personname></author>
  </authorgroup>
  <title>A book with a list of one author</title>
</biblioentry>

<biblioentry>
  <authorgroup>
  <author><personname>Jane Smith</personname></author>
  <author><personname>John Smith</personname></author>
  </authorgroup>
  <title>A book with two authors</title>
</biblioentry>

<biblioentry>
  <authorgroup>
  <author><personname>Jane Smith</personname></author>
  <author><personname>John Smith</personname></author>
  <author><personname>Sarah Smith</personname></author>
  </authorgroup>
  <title>A book with three authors</title>
</biblioentry>

<biblioentry>
  <authorgroup>
  <author><personname>Jane Smith</personname></author>
  <author><personname>John Smith</personname></author>
  <author><personname>Sarah Smith</personname></author>
  <author><personname>John Doe</personname></author>
  </authorgroup>
  <title>A book with four authors</title>
</biblioentry>

<biblioentry>
  <authorgroup>
  <author><personname>Jane Smith</personname></author>
  <author><personname>John Smith</personname></author>
  <author><personname>Sarah Smith</personname></author>
  <author><personname>John Doe</personname></author>
  <author><personname>John Bigboote</personname></author>
  </authorgroup>
  <title>A book with five authors</title>
</biblioentry>

</bibliolist>

</article>
