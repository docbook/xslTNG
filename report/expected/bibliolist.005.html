<!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" class="no-js"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><title>Unit Test: bibliolist.005</title><meta name="viewport" content="width=device-width, initial-scale=1.0"/><link href="https://purl.org/dc/elements/1.1/" rel="schema.dc"/><meta content="2011-04-22T17:02:00-06:00" name="dc.modified"/><meta content="DocBook xslTNG" name="generator"/><link href="./css/docbook.css" rel="stylesheet" media="screen"/></head><body class="home"><nav class="top"></nav><main><article class="article component"><header><h1>Unit Test: bibliolist.005</h1></header><p>Tests for lists of names.</p><div class="bibliolist">
<p class="biblioentry"><cite class="title">A book with one author</cite>. <span class="author"><span class="first-last personname">Jane Smith</span></span>. </p>

<p class="biblioentry"><cite class="title">A book with a list of one author</cite>. <span class="authorgroup"><span class="author"><span class="first-last personname">Jane Smith</span></span></span>. </p>

<p class="biblioentry"><cite class="title">A book with two authors</cite>. <span class="authorgroup"><span class="author"><span class="first-last personname">Jane Smith</span></span> and <span class="author"><span class="first-last personname">John Smith</span></span></span>. </p>

<p class="biblioentry"><cite class="title">A book with three authors</cite>. <span class="authorgroup"><span class="author"><span class="first-last personname">Jane Smith</span></span>, <span class="author"><span class="first-last personname">John Smith</span></span>, and <span class="author"><span class="first-last personname">Sarah Smith</span></span></span>. </p>

<p class="biblioentry"><cite class="title">A book with four authors</cite>. <span class="authorgroup"><span class="author"><span class="first-last personname">Jane Smith</span></span>, <span class="author"><span class="first-last personname">John Smith</span></span>, <span class="author"><span class="first-last personname">Sarah Smith</span></span>, and <span class="author"><span class="first-last personname">John Doe</span></span></span>. </p>

<p class="biblioentry"><cite class="title">A book with five authors</cite>. <span class="authorgroup"><span class="author"><span class="first-last personname">Jane Smith</span></span>, <span class="author"><span class="first-last personname">John Smith</span></span>, <span class="author"><span class="first-last personname">Sarah Smith</span></span>, <span class="author"><span class="first-last personname">John Doe</span></span>, and <span class="author"><span class="first-last personname">John Bigboote</span></span></span>. </p>

</div></article></main><nav class="bottom"></nav></body></html>